### DISCUSSION TITLE
Translation: ukrainian

### DISCUSSION THREAD

**Discussion by kopach** [rxhanson/Rectangle#1094] [Ideas]
Hi @rxhanson, I'm native Ukrainian speaker and can help to add Ukrainian translation to this great app. I'm not MacOs dev. Could you, please, help to prepare translation files for me, and I'll provide the translations itself? I'll start translation work in meanwhile

**Comment by rxhanson** [comment]
Thanks for looking to contribute!

I went ahead and generated the file, here it is: 

https://github.com/rxhanson/Rectangle/blob/master/Rectangle/uk.lproj/Main.strings

You can make a pull request with your changes, or just drop the file here and I can add it in.

**Reply by kopach** [reply]
Thanks. Here is PR https://github.com/rxhanson/Rectangle/pull/1095
Btw, how about this file https://github.com/rxhanson/Rectangle/blob/aa434500ec61292c5bf7da92c8fb0de477abdf54/RectangleLauncher/uk.lproj/Main.strings 
Should it also be translated?
